dbtmetabase/dbt.py

```diff
@@ -36,12 +36,23 @@ def read_models(self, includes=[], excludes=[]) -> list:
         mb_models = []
 
         for path in (Path(self.project_path) / "models").rglob("*.yml"):
+            logging.info(path)
             with open(path, "r") as stream:
                 schema = yaml.safe_load(stream)
+                if schema is None:
+                    logging.warn(f"SKIPPING EMPTY/INVALID YML: {path}")
```

Comment:
Please don't use all-caps in logs, it's inconsistent with other logs. Also no need for f-strings in logging statements, it takes arguments:

```
logging.warn("Skipping empty or invalid YAML: %s", path)
```